es6-lib/decoders/layer.js

```diff
@@ -95,6 +95,8 @@ class Layer extends Duplex {
   }
 
   belongsIn(soqlRow) {
+    if(soqlRow.length !== this.columns.length) return false;
```

Comment:
I feel like you should put curly braces around this because it's side-effecting and is easy to get lost.

```js
if (soqlRow.length !== this.columns.length) {
  return false;
}
```